firmware/application/apps/ui_settings.cpp

```diff
@@ -726,6 +730,37 @@ void SetConfigModeView::focus() {
     button_save.focus();
 }
 
+/* FakeBrightnessView ************************************/
+
+SetFakeBrightnessView::SetFakeBrightnessView(NavigationView& nav) {
+    add_children({&labels,
+                  &field_fake_brightness,
+                  &button_save,
+                  &button_cancel,
+                  &checkbox_brightness_switch});
+
+    field_fake_brightness.set_by_value(pmem::fake_brightness_level());
+    checkbox_brightness_switch.set_value(pmem::apply_fake_brightness());
+
+    checkbox_brightness_switch.on_select = [this](Checkbox&, bool v) {
+        pmem::set_ui_hide_fake_brightness(v ? false : true);  // if user disabled fake brightness, we don't need icon to confusing user.
```

Comment:
Will check if this is equal to !v when tomorrow I’m sober😅

Reply:
It is.  :-)

Reply:
fixed the logic.